deepdrivemd/sim/openmm_stream/run_openmm.py

```diff
@@ -0,0 +1,207 @@
+import shutil
+import simtk.unit as u
+import simtk.openmm as omm
+from mdtools.openmm.sim import configure_simulation
+from deepdrivemd.utils import Timer, parse_args
+from deepdrivemd.sim.openmm_stream.config import OpenMMConfig
+from deepdrivemd.sim.openmm_stream.openmm_reporter import ContactMapReporter
+import sys
+import os
+import time
+import random
+from lockfile import LockFile
+import pickle
+import parmed as pmd
+import numpy as np
+import subprocess
+import itertools
+from deepdrivemd.sim.openmm.run_openmm import SimulationContext
+
+
+def configure_reporters(
+    sim: omm.app.Simulation,
+    ctx: SimulationContext,
+    cfg: OpenMMConfig,
+    report_steps: int,
+):
+
+    sim.reporters.append(ContactMapReporter(report_steps, cfg))
+
```

Comment:
In this file, several variables need to be added to the `deepdrivemd.sim.openmm_stream.config.OpenMMConfig`. They are: `next_outlier_policy`, `outliers_dir`, `pickle_db`, `copy_velocities_p`.

Also is it possible to change `next_outlier_policy` from an integer to a string type with a descriptive name?

Reply:
Done. For now next_outlier_policy variable is removed since only one policy is used so far.